test/t8_test_cmesh_partition.cxx

```diff
@@ -111,75 +113,48 @@ test_cmesh_partition_concentrate (t8_cmesh_t cmesh_partition_uniform,
 }
 
 static void
-test_cmesh_partition (sc_MPI_Comm comm)
+test_cmesh_partition (int cmesh_id, sc_MPI_Comm comm)
 {
-  int                 eci, level, maxlevel, minlevel;
+  int                 level = 11;
   int                 mpisize, mpiret;
   int                 i;
   t8_cmesh_t          cmesh_original, cmesh_partition;
 
   mpiret = sc_MPI_Comm_size (comm, &mpisize);
   SC_CHECK_MPI (mpiret);
+  cmesh_original = t8_test_create_cmesh (cmesh_id);
+
+  test_cmesh_committed (cmesh_original);
+  for (i = 0; i < 2; i++) {
+    /* Set up the partitioned cmesh */
+    t8_cmesh_init (&cmesh_partition);
+    t8_cmesh_set_derive (cmesh_partition, cmesh_original);
+    /* Uniform partition according to level */
+    t8_cmesh_set_partition_uniform (cmesh_partition, level,
+                                    t8_scheme_new_default_cxx ());
+    t8_cmesh_commit (cmesh_partition, comm);
+    test_cmesh_committed (cmesh_partition);
+    cmesh_original = cmesh_partition;
+  }
+  /* Perform the concentrate test */
+  test_cmesh_partition_concentrate (cmesh_partition, comm, level);
+  /* Clean-up */
+  t8_cmesh_destroy (&cmesh_partition);
+}
+
+/* The function test_cmesh_copy_all(sc_MPI_Comm comm) runs the cmesh_copy test for all cmeshes we want to test.
+ * We run over all testcases using t8_get_all_testcases() to know how many to check. 
+ * \param [in] comm The communicator used in test_cmesh_copy to commit the cmesh copy.
+ */
+static void
+test_cmesh_partition_all (sc_MPI_Comm comm)
+{
+  /* Test all cmeshes over all different inputs we get through their id */
+  for (int cmesh_id = 0; cmesh_id < t8_get_number_of_all_testcases ();
+       cmesh_id++) {
+    t8_debugf ("Cmesh_id=%i \n", cmesh_id);
```

Comment:
This console-output is fine for debugging, but should be removed eventually, since we print out the name of every mesh before testing it